app/assets/stylesheets/orders.css.scss

```diff
@@ -22,14 +22,94 @@
   }
 }
 
-.payment-form {
-  width: 30%;
+.checkout {
+  @include span-columns(6);
 
-  .alert {
-    width: 950px;
+  #{$all-text-inputs} {
+    background: transparent;
+    border: none;
+    box-shadow: none;
+    height: auto;
+    line-height: 1.5em;
+    margin: 0;
+    padding: 1.25em 0;
+    width: 65%;
   }
 
-  fieldset {
-    background: none;
+  h5 {
+    text-transform: uppercase;
+  }
+
+  label {
+    color: #555;
```

Comment:
Color literals like `#555` should only be used in variable declarations; they should be referred to via variable everywhere else.